src/System.Net.Sockets/ref/System.Net.Sockets.cs

```diff
@@ -88,17 +88,27 @@ public partial class NetworkStream : System.IO.Stream
     {
         public NetworkStream(System.Net.Sockets.Socket socket) { }
         public NetworkStream(System.Net.Sockets.Socket socket, bool ownsSocket) { }
+        public NetworkStream(System.Net.Sockets.Socket socket, System.IO.FileAccess access) { }
+        public NetworkStream(System.Net.Sockets.Socket socket, System.IO.FileAccess access, bool ownsSocket) { }
         public override bool CanRead { get { return default(bool); } }
         public override bool CanSeek { get { return default(bool); } }
         public override bool CanTimeout { get { return default(bool); } }
         public override bool CanWrite { get { return default(bool); } }
         public virtual bool DataAvailable { get { return default(bool); } }
         public override long Length { get { return default(long); } }
         public override long Position { get { return default(long); } set { } }
+        protected bool Readable { get { return default(bool); } set { } }
         public override int ReadTimeout { get { return default(int); } set { } }
+        protected Socket Socket { get { return default(Socket); } }
+        protected bool Writeable { get { return default(bool); } set { } }
         public override int WriteTimeout { get { return default(int); } set { } }
+        public IAsyncResult BeginRead(byte[] buffer, int offset, int size, AsyncCallback callback, Object state) { return default(IAsyncResult); }
```

Comment:
I remember we had testing for APM that got removed at some point. (Didn't look - maybe the code is still there?).
The tests should be in older versions of the repo and can easily be brought back (I believe they were in the System.Net.Sockets.Legacy folder.)